routing_common/transit_types.hpp

```diff
@@ -218,6 +216,28 @@ class Gate
   m2::PointD m_point;
 };
 
+class ShapeId
+{
+public:
+  ShapeId() = default;
+  ShapeId(StopId stop1_id, StopId stop2_id) : m_stop1Id(stop1_id), m_stop2Id(stop2_id) {}
```

Comment:
Why you don't follow our style guide for namings? ```stop1Id```, ```stop2Id```. Or, you may shorten names here:
```c++
ShapeId(StopId id1, StopId id2)
```

Reply:
Yes. id1, id2.